nearswap/src/lib.rs

```diff
@@ -817,36 +818,36 @@ mod tests {
 
         ctx.vm.predecessor_account_id = ctx.accounts.owner;
         testing_env!(ctx.vm);
-        let owner2 = "new_owner_near".to_string();
-        c.change_owner(owner2.clone());
-        assert_eq!(c.owner, owner2);
+
+        c.change_owner("new_owner_near".try_into().unwrap());
+        assert_eq!(c.owner, "new_owner_near");
     }
 
     #[test]
     #[should_panic(expected = "Only the owner can call this function")]
     fn change_owner_other_account() {
         let (_, mut c) = init();
-        let owner2 = "new_owner_near".to_string();
+        let owner2: ValidAccountId = "new_owner_near".try_into().unwrap();
```

Comment:
This we could write:
```
let owner2 = ValidAccountId::from("new_owner_near");
// or:
let owner2: ValidAccountId = ("new_owner_near").into();
```

Reply:
![from](https://user-images.githubusercontent.com/26249903/109804628-75c0ef00-7c48-11eb-952a-2435ea0bf495.png)

I am not able to use it, i am getting this error.

Reply:
Yes, `ValidAccountId` doesn't implement `From` for `string`. It only implements `TryFrom`.